app/js/actions/createLabOrder.js

```diff
@@ -0,0 +1,185 @@
+import 'babel-polyfill';
+import randomString, { generate } from 'randomstring';
+
+import loading from './loading';
+import axiosInstance from '../config';
+
+import {
+  CREATE_LAB_ORDER,
+  CREATE_LAB_ORDER_SUCCESS,
+  CREATE_LAB_ORDER_FAILURE,
+} from './actionTypes';
+
+const createLabOrderFailure = error => ({
+  type: CREATE_LAB_ORDER_FAILURE,
+  error,
+});
+
+const createLabOrderSuccess = data => ({
+  type: CREATE_LAB_ORDER_SUCCESS,
+  data,
+});
+
+const createLabOrder = labOrderData => async (dispatch) => {
+  try {
+    dispatch(loading(CREATE_LAB_ORDER, true));
+    const requestPayload = {
+      encounterProviders: [
+        {
+          encounterRole: '240b26f9-dd88-4172-823d-4a8bfeb7841f',
+          provider: 'f9badd80-ab76-11e2-9e96-0800200c9a66',
```

Comment:
These should come from variables, like they do for drug order... if you are hardcoding these for testing, that's fine, but it may not work on another users system:

https://github.com/openmrs/openmrs-owa-orderentry/blob/master/app/js/components/drugOrderEntry/addForm/DraftDataTable.jsx#L93